streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -663,7 +666,6 @@ public void sendShutdownRequest(final AssignorError assignorError) {
         log.warn("Detected that shutdown was requested. " +
```

Comment:
nit: move this to where we call `enforceRebalance` as well

Reply:
done